cmd/snapctl/task.go

```diff
@@ -343,6 +339,11 @@ func createTaskUsingTaskManifest(ctx *cli.Context) error {
 		return fmt.Errorf("Unsupported file type %s\n", ext)
 	}
 
+	// Validate task manifest includes schedule, workflow, and version
+	if err := validateTask(t); err != nil {
+		return err
+	}
+
```

Comment:
The idea is to validate that both a schedule and workflow were included in the task manifest file and throw an appropriate error before the code attempts to use certain aspects of the created task struct so that the user will receive a better error than `runtime error: invalid memory address or nil pointer dereference`.